tests/fail/elaboration/non-exhaustive-patterns/match-duplicate.snap

```diff
@@ -0,0 +1,19 @@
+stdout = ''
+stderr = '''
+warning: unreachable pattern
+  ┌─ tests/fail/elaboration/non-exhaustive-patterns/match-duplicate.fathom:5:3
+  │
+5 │   true => 2
+  │   ^^^^
+
+error: non-exhaustive patterns in match expression
+  ┌─ tests/fail/elaboration/non-exhaustive-patterns/match-duplicate.fathom:3:7
+  │  
+3 │ ╭ match true {
+  │         ^^^^ patterns not covered
```

Comment:
This is a bug in codespan. I'm pretty sure it was fixed but we might not have deployed it to crates.io

Reply:
What's buggy about it?

Reply:
It's missing the vertical line in the gutter

Reply:
```
  ┌─ tests/fail/elaboration/non-exhaustive-patterns/match-duplicate.fathom:3:7
  │  
3 │ ╭ match true {
  │ │       ^^^^ patterns not covered
  ```